README

```diff
@@ -7,15 +7,23 @@ This package and the code it generates requires at least Go 1.4.
 This software implements Go bindings for protocol buffers.  For
 information about protocol buffers themselves, see
 	https://developers.google.com/protocol-buffers/
-To use this software, you must first install the standard C++
-implementation of protocol buffers from
+
+## Installation ##
+
+To use this software, you must:
+- Install the standard C++ implementation of protocol buffers from
 	https://developers.google.com/protocol-buffers/
-And of course you must also install the Go compiler and tools from
-	https://golang.org/
-See
-	https://golang.org/doc/install
-for details or, if you are using gccgo, follow the instructions at
-	https://golang.org/doc/install/gccgo
+- Of course, install the Go compiler and tools from https://golang.org/
+	See
```

Comment:
please use two spaces instead of a tab for these sentences, so that they are in line with the bullets.
continue to use a tab for indenting the URLs and commands